cross/sshfs/Makefile

```diff
@@ -0,0 +1,17 @@
+PKG_NAME = sshfs
+PKG_VERS = 2.5
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-fuse-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://sourceforge.net/projects/fuse/files/$(PKG_NAME)-fuse/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-fuse-$(PKG_VERS)
+
+DEPENDS = cross/fuse cross/glib
+
+HOMEPAGE = http://fuse.sourceforge.net/sshfs.html
+COMMENT  = SSHFS is a filesystem client based on the SSH File Transfer Protocol.
```

Comment:
From what I've seen from the documentation, I believe the name of the client is referred to as `sshfs` (generally all lower case)
`SSHFS` is used for the actual filesystem itself. Can you verify that?